chstrings/qqq.json

```diff
@@ -11,7 +11,7 @@
 	"back": "{{Identical|Back}}",
 	"button_next": "Label on the button that browses to the next article.\n{{Identical|Next}}",
 	"button_wikilink": "Label on the button that links to the article so the user can add a reference.",
-	"category_filter_placeholder": "The placeholder text in the input box for filtering articles by category.",
+	"category_filter_placeholder": "The placeholder text in the input box for filtering articles by category",
```

Comment:
I guess this one can stay, right? It's not shown anywhere, it's just a description of what the string is for translators.

Reply:
Oh yes, sorry, that got scooped up by my overzealous regex search; I'll restore the period.

Reply:
Actually, if this is used in a form-field `placeholder` value, it should not end in a period. Where does this text show up?

Reply:
It's only visible in TranslateWiki, the tool that volunteers use to translate these strings. It is an explanation (to the translator) of where the string will eventually be used, so it's not used in an actual form. See [here](https://translatewiki.net/w/i.php?title=Special:Translate&showMessage=citationhunt-category_filter_placeholder&group=citationhunt&language=fr&filter=&optional=1&action=translate) for a concrete example.